pkg/config/config_template.yaml

```diff
@@ -2520,6 +2520,15 @@ api_key:
 #
 # kubernetes_https_kubelet_port: 10250
 
+## @param kubernetes_distribution - string - optional - default: ""
+## @env DD_KUBERNETES_DISTRIBUTION - string - optional - default: ""
+# The Kubernetes distribution where the agent is deployed. This automatically
+# applies options needed to make the agent run correctly on each distribution.
+# For now, this is only needed when running on AKS. When that's the case, set
+# this option to "aks".
+#
```

Comment:
What would it be set to in other circumstances?  It seems like this has been added as a way to say "should we apply the AKS-specific workaround" -- maybe a more specific name would be better?

Reply:
Right. I'll change this. I got similar feedback from @vboulineau

Reply:
I added a new commit to change this: https://github.com/DataDog/datadog-agent/pull/11367/commits/a51cb37631c82b3d4473e0cca1feb55275705f0b